src/components/home/elements/SendToken.vue

```diff
@@ -624,17 +630,35 @@ export default {
             return common.getLength(this.description) <= TRANSFER_ATTACHMENT_BYTE_LIMIT
         },
         getAddressFromDataObject() {
-            return this.dataObject.stored_tx.senderPublicKey ? this.account.convertPublicKeyToAddress(this.dataObject.stored_tx.senderPublicKey, this.dataObject.network_byte) : this.coldAddress
+            try {
+                return this.account.convertPublicKeyToAddress(this.dataObject.stored_tx.senderPublicKey, this.dataObject.network_byte)
+            } catch (e) {
+
+            }
+        },
+        isValidColdPublicKey() {
```

Comment:
it seems better to use "isValidPublicKey" since it also checked the case of 'hotWallet" and used it in `isSubmitDisabled`.